packages/node_modules/@node-red/editor-client/src/js/ui/typeSearch.js

```diff
@@ -382,7 +382,7 @@ RED.typeSearch = (function() {
         var items = [];
         RED.nodes.registry.getNodeTypes().forEach(function(t) {
             var def = RED.nodes.getType(t);
-            if (def.category !== 'config' && t !== 'unknown' && t !== 'tab') {
+            if (def.set.enabled && def.category !== 'config' && t !== 'unknown' && t !== 'tab') {
```

Comment:
```suggestion
            if (def.set?.enabled !== false && def.category !== 'config' && t !== 'unknown' && t !== 'tab') {
```

Subflows will show up in the result of `getNodeTypes` and they dont have the `set` property in their definition. So need to guard against that.